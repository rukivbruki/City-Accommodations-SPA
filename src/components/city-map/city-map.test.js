import React from 'react';
import renderer from 'react-test-renderer';
import CityMap from './city-map.jsx';
import mockPlaces from '../../mocks/mock-offers.js';
import mockCities from '../../mocks/mock-cities.js';
import fixContainerLeafletTest from '../../utils/fix-container-leaflet-test';

it(`CityMap snapshot`, () => {
  fixContainerLeafletTest();
  const tree = renderer.create(<CityMap places={mockPlaces} city={mockCities[0]} />).toJSON();
  expect(tree).toMatchSnapshot();
});
