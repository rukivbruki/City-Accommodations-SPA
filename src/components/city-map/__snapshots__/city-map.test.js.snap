// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityMap snapshot 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
>
  <div />
</div>
`;
